src/lightning_lite/lite.py

```diff
@@ -489,10 +554,37 @@ def _validate_setup(model: nn.Module, optimizers: Sequence[Optimizer]) -> None:
         if any(isinstance(opt, _LiteOptimizer) for opt in optimizers):
             raise ValueError("An optimizer should be passed only once to the `setup` method.")
 
+    def _validate_setup_model(self, model: nn.Module) -> None:
```

Comment:
Error messaging here for built-in strategies provide guidance to users who need to change their code to call the setup methods in the right way.